doc/src/manual/ranch.info.asciidoc

```diff
@@ -38,9 +38,11 @@ transport:: Transport module.
 transport_options:: Transport options.
 protocol:: Protocol module.
 protocol_options:: Protocol options.
+metrics:: Listener metrics.
```

Comment:
You should describe what metrics are available. Maybe add a section `== Metrics` right after this to list the metric keys.

Reply:
See last commit.